### PR TITLE
Show analysis period in contribution bar tooltip #362

### LINKED ISSUE #362: Show analysis period in contribution bar tooltip
Current: contribution bar tooltip shows the total contribution

Suggested: Show `Total contribution from dd-mm-yyyy to dd-mm-yyyy: nnn lines`

### PR DESCRIPTION
**Pull request by apoorva17**
Fixes #362 
```
Tooltip shows only total contribution.

Let's include the analysis period for the total contribution in the
tooltip to make it more user-friendly.
```

### CODE REVIEW

**Comment by eugenepeh** [comment]
image

Bug occurs when revisiting using bookmarked url.

Putting this on hold as there's some dependency on #359

**Comment by apoorva17** [comment]
@eugenepeh issue #359 seems to be resolved, can the hold be removed?

**Comment by eugenepeh** [comment]
> @eugenepeh issue #359 seems to be resolved, can the hold be removed?

Oh yes... my bad. Please continue.

**Comment by apoorva17** [comment]
i've updated the branch, ready for review

**Comment by apoorva17** [comment]
@eugenepeh spaces have been added around operators

**Comment by apoorva17** [comment]
@eugenepeh spaces have been added around operators

File: frontend/src/index.jade [hunk 1]
```diff
@@ -109,7 +109,10 @@ html
                     right: (((user.commits.length-j-1)/user.commits.length) * 100) + '%'\
                   }"
                 )
-              .summary-chart__contrib(v-bind:title="'total contribution: '+user.totalCommits")
+              .summary-chart__contrib(
+                 v-bind:title=
```
Comment (eugenepeh) [inline frontend/src/index.jade]: should be 2 space indentation

File: frontend/src/index.jade [hunk 2]
```diff
@@ -109,7 +109,10 @@ html
...
+                 "'Total contribution from ' + minDate + ' to ' + maxDate + ': ' + user.totalCommits + ' lines'"
```
Comment (eugenepeh) [inline frontend/src/index.jade]: for line wrap, it should receive twice the indentation, hence 4 spaces from previous line.

File: frontend/src/index.jade (hunk 1 above)
Comment (apoorva17) [reply frontend/src/index.jade]: indentation has been updated

File: frontend/src/index.jade (hunk 2 above)
Comment (apoorva17) [reply frontend/src/index.jade]: indentation has been updated

**Comment by eugenepeh** [comment]
@damithc should we emphasize the date range as well in the code view?

**Pull request merged**